File: C/Problem0009.c
/**
 * Problem0009 - Special Pythagorean triplet
 * 
 * https://github.com/agileshaw/Project-Euler
 **/

#include <stdio.h>

int main()
{
    int i, j, k, result;

    for (i = 0; i < 1000; i++) {
        for (j = i + 1; j < 1000; j++) {
            for (k = j + 1; k < 1000; k++) {
                if ((i*i + j*j == k*k) && (i + j + k == 1000)) {
                    result = i * j * k;
                    printf("The product of the special pythagorean triplet is: %d\n", result);
                    return 0;
                }
            }
        }
    }
    
    return 0;
}

File: Python/Problem0009.py
# Problem0009 - Special Pythagorean triplet
#
# https://github.com/agileshaw/Project-Euler


def product():
     for i in range(1000):
        for j in range(i+1, 1000):
            for k in range(j+1, 1000):
                if i*i + j*j == k*k and i + j + k == 1000:
                    return i * j * k

if __name__ == "__main__":
    result = product()
    print("The product of the special pythagorean triplet is: %d" % result)
